tests/framework/PostProcessors/TemporalDataMiningPostProcessor/DimensionalityReduction/test_tdmSKLearnKernelPCA.xml

```diff
@@ -1,113 +1,114 @@
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <Models>
+    <ExternalModel ModuleToLoad="../data_generator" name="DataGeneratorModel" subType="">
+      <variables>n,Time,x1,x2,x3,x4</variables>
+    </ExternalModel>
+    <PostProcessor name="PCA" subType="DataMining">
+      <KDD lib="SciKitLearn">
+        <Features>x1,x2,x3,x4</Features>
+        <SKLtype>decomposition|KernelPCA</SKLtype>
+        <n_components>2</n_components>
+      </KDD>
+      <pivotParameter>Time</pivotParameter>
+    </PostProcessor>
+  </Models>
+
+  <Steps>
+    <MultiRun name="sampleGen">
+      <Input class="DataObjects" type="PointSet">inputPlaceHolder</Input>
+      <Model class="Models" type="ExternalModel">DataGeneratorModel</Model>
+      <Sampler class="Samplers" type="MonteCarlo">gridSampler</Sampler>
+      <Output class="DataObjects" type="HistorySet">mData</Output>
+    </MultiRun>
+    <PostProcess name="pp1" pauseAtEnd="True">
+      <Input class="DataObjects" type="HistorySet">mData</Input>
+      <Model class="Models" type="PostProcessor">PCA</Model>
+      <Output class="DataObjects" type="HistorySet">mData</Output>
+      <Output class="OutStreams" type="Print">dummy</Output>
+      <Output class="OutStreams" type="Plot">plotPCAData</Output>
+    </PostProcess>
+  </Steps>
+
+  <Distributions>
+    <Uniform name="uniDist">
+      <lowerBound>0.0</lowerBound>
+      <upperBound>150</upperBound>
+    </Uniform>
+  </Distributions>
+
+  <Samplers>
+    <Grid name="gridSampler">
+      <samplerInit>
+        <limit>150</limit>
+      </samplerInit>
+      <variable name="n">
+        <distribution>uniDist</distribution>
+        <grid construction="equal" steps="149" type="value">0.0 149.0</grid>
+      </variable>
+    </Grid>
+  </Samplers>
+
+  <OutStreams>
+    <Print name="dummy">
+      <type>csv</type>
+      <source>mData</source>
+    </Print>
+    <Plot name="plotPCAData" overwrite="True">
+      <plotSettings>
+        <gridSpace>2 1</gridSpace>
+        <plot>
+          <type>line</type>
+          <x>mData|Output|Time</x>
+          <y>mData|Output|PCADimension1</y>
+          <interpPointsX>300</interpPointsX>
+          <gridLocation>
+            <x>0</x>
+            <y>0</y>
+          </gridLocation>
+        </plot>
+        <plot>
+          <type>line</type>
+          <x>mData|Output|Time</x>
+          <y>mData|Output|PCADimension2</y>
+          <interpPointsX>300</interpPointsX>
+          <gridLocation>
+            <x>1</x>
+            <y>0</y>
+          </gridLocation>
+        </plot>
+      </plotSettings>
+      <actions>
+        <how>png</how>
+      </actions>
+    </Plot>
+  </OutStreams>
+
+  <DataObjects>
+    <PointSet name="inputPlaceHolder">
+      <Input>n</Input>
+      <Output>OutputPlaceHolder</Output>
+    </PointSet>
+    <HistorySet name="mData">
+      <Input>n</Input>
+      <Output>Time,x1,x2,x3,x4</Output>
+    </HistorySet>
+  </DataObjects>
+
+</Simulation>
```

Comment:
What happened here? Were the line endings of this file changed? I am not able to see what is different between the old and new file.